### ISSUE TITLE
Icon request: icon-yummly

### ISSUE THREAD

**Issue by oncecoupled** [FortAwesome/Font-Awesome#6138]
The latest social network for food bloggers / foodies. I'd love to use this directly, but I know some WordPress plugins use your icons, too, and it'd be nice if they could integrate this service with their own. 

Problem is that Yummly doesn't really have a great icon to use, in the first place...
Visuals: http://www.yummly.com/goodies
Current Button: https://pbs.twimg.com/profile_images/476808557044576256/GP3YFdPw.png

**Comment by amarzka** [comment]
+1

**Comment by WhiteOakCreativ** [comment]
Agreed. I'm getting more and more requests from Food Bloggers for Yummly, and I'd prefer to use Font Awesome whenever possible.

**Comment by thisisdu** [comment]
+1

**Comment by BenSibley** [comment]
+1

**Comment by tablefortwoblog** [comment]
+1

**Comment by carasmo** [comment]
+1

**Comment by oxjox** [comment]
Sorry, didn't see this one when I posted request #10821.  Attaching my files for reference.  
![64a2835c-0e95-11e7-8212-9d2831adfacd](https://cloud.githubusercontent.com/assets/26589061/24209757/8f8a878c-0efd-11e7-9efc-1e3379a55ba2.jpg)
![a5c89958-0e94-11e7-82b9-349c613fa589](https://cloud.githubusercontent.com/assets/26589061/24209758/8f8bc71e-0efd-11e7-9da2-92cc10ca0c73.png)

**Comment by skyminds** [comment]
+1, it would be great to have the yummly icon.

**Comment by RoughIndustries** [comment]
+1

**Comment by skyminds** [comment]
+1 for the Yummly icon

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
